metadeploy/api/push.py

```diff
@@ -16,6 +16,9 @@
         JOB_CANCELED
     org.:org_url
         ORG_CHANGED
+    scratch_org.:job_id
+        SCRATCH_ORG_CREATED
+        SCRATCH_ORG_ERROR
```

Comment:
@wlonk These aren't working because subscriptions to the `scratch_org` model fail the `is_known_model` check in `consumers.py`, and probably also the `has_good_permissions` check.